nltk/internals.py

```diff
@@ -540,6 +540,7 @@ def find_file_iter(
 
     # Check environment variables
     for env_var in env_vars:
+        env_var = os.path.expanduser(env_var)
```

Comment:
`env_var` is just a string like `"HUNPOS_TAGGER"`, e.g.:
https://github.com/nltk/nltk/blob/cbe715ae5b0f100a76a27fc7e035058ddffad114/nltk/tag/hunpos.py#L94
https://github.com/nltk/nltk/blob/cbe715ae5b0f100a76a27fc7e035058ddffad114/nltk/tokenize/stanford_segmenter.py#L157
https://github.com/nltk/nltk/blob/cbe715ae5b0f100a76a27fc7e035058ddffad114/nltk/parse/malt.py#L91

So, we don't want to expand these.

Reply:
Got it, thanks. Reverted this change in 9797986b6b76aaf2d724cfedfef496ba4d6a2b15.